nss-day-cohort-60/national-parks-client-v2
# Md hub page

Description:
1. Made changes to code in nav bar to make it responsive without affecting other components
2. Added Bootstrap classes and to my hub page to make it more responsive
3. Added login on enter back to login modal after recent changes to password field

Steps to test:
1. Fetch branch md-hub-page and switch to it
2. Click on My Hub in nav bar. View page as you reduce its size. Design should be responsive. 
3. While in the smallest screen size(less than 768px) in My Hub, verify that nav bar links disappears and a hamburger icon appears that can be clicked on. When the icon is clicked on, the list of nav links should appear and the carousel should also be displayed.
